redis_bitset.go

```diff
@@ -45,3 +46,16 @@ func (r *RedisBitSet) Test(offsets []uint) (bool, error) {
 
 	return true, nil
 }
+
+func (r *RedisBitSet) Delete() error {
+	keys := fmt.Sprintf("%s:*", r.keyPrefix)
+	_, err := r.conn.Do("EVAL", "return redis.call('del', unpack(redis.call('keys', ARGV[1])))", "0", keys)
+	return err
+}
+
+func (r *RedisBitSet) getKeyOffset(offset uint) (string, uint) {
+	n := uint(offset / 4e+9)
```

Comment:
I'm presuming 4e9 is the number of bits in a 512MiB string? This would look much better as a constructed const...

```go
const maxlen = 8 * 512 * 1024 * 1024
```

Or

```go
const maxlen = 8 * 512 << 20
```